packages/zsh-dropbox

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "zsh-dropbox",
+  "description": "This plugin provides 2 apps: dropbox-cli and dropbox-uploader and completions for them.",
+  "repository": "https://github.com/zpm-zsh/zsh-dropbox",
+  "author": "Grigorii Horos"
+}
```

Comment:
You haven't added a `bin`, `script` or `init` key here, so this package doesn't actually install anything.